terraform/modules/gost_website/waf.tf

```
@@ -0,0 +1,26 @@
+locals {
+  expanded_rules = flatten([
+    for r in var.managed_waf_rules : {
+      name     = r.key,
+      priority = r.priority,
+      statement = {
+        name        = r.managed_rule
+        vendor_name = "AWS"
+      },
+      visibility_config = {
+        cloudwatch_metrics_enabled = r.metric_visibility,
+        sampled_requests_enabled   = r.metric_visibility,
+        metric_name                = "${r.key}-metric"
+      }
+    }
+  ])
```

Per my [other comment](https://github.com/usdigitalresponse/usdr-gost/pull/1177/files#r1156376153):
```suggestion
  expanded_rules = flatten([
    for r in var.managed_waf_rules : {
      name     = r.key,
      priority = r.value.priority,
      statement = {
        name        = r.key
        vendor_name = "AWS"
      },
      visibility_config = {
        cloudwatch_metrics_enabled = r.value.metric_visibility,
        sampled_requests_enabled   = r.value.metric_visibility,
        metric_name                = "${r.key}-metric"
      }
    }
  ])
```